### DISCUSSION TITLE
What to set PYTHONBREAKPOINT to for remote debugging when passing arguments like "host" and "term_size"?

### DISCUSSION THREAD

**Discussion by piotr-dobrogost** [inducer/pudb#636] [Troubleshooting] [answered]
In [Remote debugging](https://documen.tician.de/pudb/starting.html#remote-debugging) section in the docs there is no information on what to set `PYTHONBREAKPOINT` to when using remote debugging. Specifically how to set its value when one wants to call `set_trace()` passing arguments like `host` and `term_size`?

**Comment by asmeurer** [comment, marked as answer]
You can set the term size with `PUDB_TERM_SIZE`. https://documen.tician.de/pudb/starting.html#remote-debugging and the host with `PUDB_RDB_HOST` (not documented, but you can see here https://github.com/inducer/pudb/blob/920414fb68aa09f4af26a967481c3a4123c77709/pudb/remote.py#L54-L56). PYTHONPBREAKPOINT can only be set to a function name, so there's no way to specify arguments with it.

**Reply by piotr-dobrogost** [reply]
> PYTHONPBREAKPOINT can only be set to a function name, so there's no way to specify arguments with it.

Thanks for information. I think this could be clearly stated in the docs so to not leave the reader wondering.

**Comment by asmeurer** [comment]
Maybe it would be useful to be able to set this in the config as well?

**Reply by piotr-dobrogost** [reply]
Sounds as a useful feature.
